app/src/main/java/fr/free/nrw/commons/upload/UploadMediaDetailAdapter.java

```diff
@@ -26,16 +26,16 @@
 import fr.free.nrw.commons.utils.BiMap;
 import timber.log.Timber;
 
-public class DescriptionsAdapter extends RecyclerView.Adapter<DescriptionsAdapter.ViewHolder> {
+public class UploadMediaDetailAdapter extends RecyclerView.Adapter<UploadMediaDetailAdapter.ViewHolder> {
```

Comment:
Seems to be related to the refactoring thing, please check

Reply:
Yes, previously there was only an adapter for descriptions since title was not multilingual. However, now We have a single adapter for both the entities description as well as captions. Hence I have renamed the corresponding class to `UploadMediaDetailAdapter`

Reply:
Okay